src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests.cs

```diff
@@ -1266,9 +1266,6 @@ public static class Extensions
 .class public auto ansi abstract sealed beforefieldinit Extensions
     extends [netstandard]System.Object
 {
-    .custom instance void [netstandard]System.Runtime.CompilerServices.ExtensionAttribute::.ctor() = (
```

Comment:
We should still emit this attribute (on the containing type). Do you mean to handle that in a later PR?